### PR TITLE
React: add Customisation namespace

### PR DESCRIPTION
**Pull request by jake-slashid**
Adds new `Customisation` namespace, with a handful of helpers for the hosted login page customisation.

New utils:
- `getGoogleFontImports`
- `sanitiseCssVariableCustomisationConfig`
- `pxValidator`
- `hexValidator`
- `exactValidator`
- `fontFamilyValidator`

### CODE REVIEW

File: packages/react/src/utils/css-sanitisation.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+export const pixelValueRegExp = /([0-9]+)px/;
+export const hexValueRegExp = /#([\da-fA-F]{2})([\da-fA-F]{2})([\da-fA-F]{2})/;
+export const fontFamilyRegExp = /(?:['"]?([\\\w\d\- ]+)['"]?(?:,\s*)?)/g;
+export const displayValues = ["flex", "none"];
+export const googleFonts = new Set(["Open Sans"])
```
Comment (ikovic) [inline packages/react/src/utils/css-sanitisation.ts:5]: Pretty sure that `Inter` is also a google font

File: packages/react/src/dev.css [hunk 2]
```diff
@@ -30,9 +30,9 @@ body {
 }
 
 .layout {
-  display: grid;
+  /* display: grid;
   grid-template-columns: 1fr 1fr;
-  gap: 1rem;
+  gap: 1rem; */
```
Comment (kaspermroz) [inline packages/react/src/dev.css:35]: Please remove commented out code

File: packages/react/src/dev.tsx [hunk 3]
```diff
@@ -163,38 +163,90 @@ const ConfiguredDynamicFlow = () => {
...
   return (
     <ConfigurationProvider
+      logo="https://membersapp.checkatrade.com/static/media/logo.e930e0049182a5b0e5bf.png"
       factors={[
-        { method: "webauthn" },
-        { method: "email_link" },
-        { method: "otp_via_email" },
-        { method: "otp_via_sms" },
-        { method: "password" },
+        // { method: "webauthn" },
```
Comment (kaspermroz) [inline packages/react/src/dev.tsx:168]: Bunch of mess in this file, I'd rather not commit these changes

**Review by kaspermroz — COMMENTED** [review]
In general I don't think we should create `otherPublicVariables` as a interface, it should be unified and with as little vars as possible to do the job

File: packages/react/src/dev.css (hunk 2 above)
Comment (ikovic) [reply packages/react/src/dev.css]: Done already in the previous commit

File: packages/react/src/utils/css-sanitisation.ts (hunk 1 above)
Comment (ikovic) [reply packages/react/src/utils/css-sanitisation.ts]: Fixed!

File: packages/react/src/dev.tsx (hunk 3 above)
Comment (ikovic) [reply packages/react/src/dev.tsx:168]: None of these will be committed, I'll clean this up after I'm done testing

**Review by kaspermroz — APPROVED** [review]
Great job guys! 🎖️

**Pull request merged**